tests/test_rest_server.py

```diff
@@ -238,26 +238,20 @@ def test_boolify():
         'claimant': 'testing-picker-aaaed864-0112-4bcf-a069-bb55c12e291d',
     }
     ret = await r.request('POST', '/TransferRequests/actions/pop?source=NERSC', nersc_pop_claimant)
-    assert ret['results'] == []
+    assert ret['transfer_request'] is None
```

Comment:
Instead of asserting None, it might be better to just assert the negation:
`assert not ret['transfer_request']`
This is more future proof in case we decide it should be an empty dict or something else in the future.  Also falls into python's duck typing ethos.

Reply:
Fixed up all 8 of these in `test_rest_server.py`